documentation/docs/guides/Getting Started/What is Electrobun.md

```diff
@@ -4,21 +4,21 @@ title: What is Electrobun
 sidebar_label: 1. What is Electrobun
 ---
 
-Electrobun aims to be a batteries included framework for building desktop apps with Typescript.
+Electrobun aims to be a battery-included framework for building desktop apps with Typescript.
 
 ## Priorities
 
-What makes electrobun different, aside from our approach to achitecture are our project goals:
+What makes electrobun different, aside from our approach to architecture are our project goals:
```

Comment:
I agree with this change, and (wasting -> wasiting which was already fixed in another PR), but disagree with the other changes